src/components/AccessibilityDocumentsList/index.tsx

```diff
@@ -23,24 +37,69 @@ type Document = {
 type DocumentsListProps = {
   documents: Document[];
   requestName: string;
+  requestId: string;
 };
 
 const AccessibilityDocumentsList = ({
   documents,
-  requestName
+  requestName,
```

Comment:
What are we using these additional props for? I don't see them, but I might be missing it.

Reply:
requestName is used in the modal, and requestID is used in the mutation